### PR TITLE
Added fun/dfu

### PR DESCRIPTION
**Pull request by YTG1234**
Thank you to @ramidzkh for making that meme.

### CODE REVIEW

**Review by Juuxel — APPROVED** [review]

File: tags/fun/dfu.ytag [hunk 1]
```diff
@@ -0,0 +1,7 @@
+type: text
+
+---
+
+<:lambdasoup:798677550159102012><:steve_lambda:691095252458537011><:lambdasoup2:798683897730564136>
+
+https://youtu.be/lClqn2k6W_4
```
Comment (ramidzkh) [inline tags/fun/dfu.ytag:7]: no use https://youtu.be/ON1vN9NnCc8

Comment (YTG1234) [reply tags/fun/dfu.ytag:7]:
```suggestion
https://youtu.be/ON1vN9NnCc8
```
Just putting this here so I can add it quickly

**Review by ramidzkh — APPROVED** [review]
I see no flaws whatsoever

**Review by CoolMineman — APPROVED** [review]

**Review by hYdos — APPROVED** [review]

**Review by i509VCB — APPROVED** [review]
Glorious.

**Pull request merged**